adhocracy4/comments_async/static/comments_async/comment_box.jsx

```diff
@@ -62,7 +62,7 @@ export default class CommentBox extends React.Component {
       showFilters: false,
       filter: 'all',
       filterDisplay: django.gettext('all'),
-      sort: 'new',
+      sort: props.useModeratorMarked ? 'mom' : 'new',
```

Comment:
So, this is also the default filter when it is used?

Reply:
Yes. I thought we can also change that once we use it somewhere else, where it is not supposed to be default? That would be another setting, I guess? Or the setting would be a tuple?

Reply:
On the tests btw..

Reply:
Yeah, we'll worry about that when we get there. It might just be default and have the same wording anyway. :)